CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 ## Changelog
 
+### `1.3.0`
```

Comment:
I wasn't sure whether to make this a patch release or a minor release since previous attempts to enforce TLS 1.2 were already made. Open to suggestions.

Reply:
Patch seems reasonable. 👍